docs/validation.md

```diff
@@ -12,34 +12,34 @@ and checks that those files are correct and can be used with the App Orchestrati
 You can run all at once or separated.
 
 ### Run all validations at once
-`vendor/bin/app-sdk validate:app`
+`vendor/bin/app-sdk app:validate`
 
 #### Configuration options
 
 The command offers the following configuration.
 
-- `vendor/bin/app-sdk validate:app -m path/to/manifest/files`
-- `vendor/bin/app-sdk validate:app -c path/to/configuration/file/configuration.json`
-- `vendor/bin/app-sdk validate:app -t path/to/translation/file/translation.json`
-- `vendor/bin/app-sdk validate:app -a path/to/asyncapi/file/asyncapi.yml`
+- `vendor/bin/app-sdk app:validate -m path/to/manifest/files`
+- `vendor/bin/app-sdk app:validate -c path/to/configuration/file/configuration.json`
+- `vendor/bin/app-sdk app:validate -t path/to/translation/file/translation.json`
+- `vendor/bin/app-sdk app:validate -a path/to/asyncapi/file/asyncapi.yml`
 
 ### Run manifest file validation
-`vendor/bin/app-sdk validate:manifest`
+`vendor/bin/app-sdk app:manifest:validate`
 
 #### Configuration options
 
 The command offers the following configuration.
 
-`vendor/bin/app-sdk validate:manifest -m path/to/manifest/files`
+`vendor/bin/app-sdk app:manifest:validate -m path/to/manifest/files`
```

Comment:
What kind of manifests are those? The recipes for integrator?

Reply:
App manifests. Each App has those files which define basic information we need to display them in an AppStoreCatalog

Reply:
I see, thx.